modules/visual_script/doc_classes/VisualScriptCustomNode.xml

```diff
@@ -131,7 +131,7 @@
 				The [code]inputs[/code] array contains the values of the input ports.
 				[code]outputs[/code] is an array whose indices should be set to the respective outputs.
 				The [code]start_mode[/code] is usually [constant START_MODE_BEGIN_SEQUENCE], unless you have used the [code]STEP_*[/code] constants.
-				[code]working_mem[/code] is an array which can be used to persist information between runs of the custom node.
+				[code]working_mem[/code] is an array which can be used to persist information between runs of the custom node. The size needs to be predefined using [method _get_working_memory_size]. Do not overwrite [code]working_mem[/code] using a other [code]var[/code].
```

Comment:
> using a other

Can you check the grammar and spelling?

Reply:
Beter?

Reply:
> using a other

using another [code]var[/code].

Reply:
Ya, I installed Grammarly to help me.
![image](https://user-images.githubusercontent.com/20573784/130502394-438a7ed3-a07a-4020-b907-f64fb0404f79.png)